ood-portal-generator/templates/ood-portal.conf.erb

```diff
@@ -230,6 +230,29 @@ Listen <%= addr_port %>
     <%= line %>
     <%- end -%>
 
+    <%- if @oidc_uri && @oidc_provider_metadata_url && @oidc_client_id -%>
+    # Fix Cross-site request proxying autentication data leaks
+    RequestHeader edit* Cookie "mod_auth_openidc_session_\d=[^;]+;" ""
+    RequestHeader unset OIDC_CLAIM_sub
```

Comment:
I'm not going to highlight all of these - but this comment applies: this should likely come from some helper function so it's not repeated and can be updated in 1 place instead of 2.